packages/playground/src/components/node_selector/TfNodeDetailsCard.vue

```diff
@@ -466,7 +503,35 @@ export default {
       }
     }
 
+    const discountPackagesItems = [
+      { name: "Default", discount: "-20%", tfts: 200 },
+      { name: "Bronze", discount: "-30%", tfts: 400 },
+      { name: "Silver", discount: "-40%", tfts: 600 },
+      { name: "Gold", discount: "-60%", tfts: 1000 },
```

Comment:
The TFTs required to unlock the discount should be calculated based on the node resources.